nvtabular/loader/backend.py

```diff
@@ -504,4 +524,39 @@ def _create_tensors(self, gdf):
         return tensors
 
     def _handle_tensors(self, cats, conts, labels):
-        return cats, conts, labels
+        X = {}
+        for tensor, names in zip([cats, conts], [self.cat_names, self.cont_names]):
+            lists = {}
+            if isinstance(tensor, tuple):
+                tensor, lists = tensor
+            names = [i for i in names if i not in lists]
+
+            # break list tuples into two keys, with postfixes
+            # TODO: better choices for naming?
+            list_columns = list(lists.keys())
+            for column in list_columns:
+                values, nnzs = lists.pop(column)
+                lists[column] = values, nnzs
```

Comment:
Umm - is this necessary? It seems like you're popping of a tuple of (values, nnzs) from column and then inserting back with the same key - can we just delete?

```suggestion
```

Reply:
Ah, maybe this chunk of code was intended to do the rename mentioned in the `TODO` comment above? This would be a reasonable way to do that if the postfixes were applied here.

Reply:
it was used to break apart the lists into values and nnz dictionary entries.